products-dashboard/products-dashboard.dash.yaml

```diff
@@ -0,0 +1,47 @@
+global:
+  mode: dark
+  settings:
+    chart:
+      resizable: true 
+datasets:
+  - uuid: products
+    content: >-
+      [
+        ["Computers", "Scanner", 5, 3],
+        ["Computers", "Printer", 7, 4],
+        ["Computers", "Laptop", 3, 2],
+        ["Electronics", "Camera", 10, 7],
+        ["Electronics", "Headphones", 5, 9]
+      ]
+    columns:
+      - id: Section
+        type: LABEL
+      - id: Product
+        type: LABEL
+      - id: Quantity
+        type: NUMBER
+      - id: Quantity2
+        type: NUMBER
+pages:
+  - components:
+      - html: Welcome to Dashbuilder!
+        properties:
+          font-size: xx-large
+          margin-bottom: 30px
+      - settings:
+          type: BARCHART
+          lookup:
+            uuid: products
+            group:
+              - columnGroup:
+                  source: Product
+                functions:
+                  - source: Product
+                  - source: Quantity
+                    function: SUM
+                  - source: Quantity2
+                    function: SUM
+      - settings:
+          type: TABLE
```

Comment:
You can remove this line, the type is not required for tables - it is the default type